platformpolicy/internal/sign/wrapper.go

```diff
@@ -26,14 +26,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+const BigIntLength = 32
```

Comment:
BigIntLength might be not exported - bigIntLength, coz it uses only in this package

Reply:
done